src/sidebar/components/tag-editor.js

```diff
@@ -87,96 +95,90 @@ function TagEditor({ onEditTags, tags: tagsService, tagList }) {
       return;
     }
     updateTags([...tagList, value]);
-    setShowSuggestions(false);
+    setOpen(false);
 
     inputEl.current.value = '';
     inputEl.current.focus();
   };
 
   /**
-   *  If the user pressed enter or comma while focused on
-   *  the <input>, then add a new tag.
+   *  Update the suggestions if the user changes the value of the input
    */
-  const handleKeyPress = e => {
-    if (e.key === 'Enter' || e.key === ',') {
-      addTag();
-      // preventDefault stops the delimiter from being
-      // added to the input field
-      e.preventDefault();
-    }
-  };
-
   const handleOnInput = e => {
```

Comment:
handleOnInput is now very simple and is only checking for changes to the text value in the input field, the other stuff is now handled all in keyDown because we have to have keyDown handler to check for up/down key navigation anyway and it just made more sense to move the "Enter" AND "," checking there to add a tag.